src/igvfd/searches/configs/construct_library.py

```diff
@@ -47,7 +72,9 @@ def cell_line():
             },
             'reporter_library_details': {
                 'title': 'Reporter Library Details'
-            }
+            },
+            'expression_library_details': {
```

Comment:
Just missing the word 'vector' here, should be "expression_vector_library_details"
Also adds a trailing comma

Reply:
thanks for catching that. Should be fixed now.